envsec/internal/flow/init.go

```diff
@@ -0,0 +1,199 @@
+package flow
```

Comment:
initflow?

Reply:
I was thinking we can use this for more than just init?